files/image_config/fstrim/fstrim.service

```diff
@@ -3,4 +3,4 @@ Description=Discard unused blocks
 
 [Service]
 Type=oneshot
-ExecStart=/sbin/fstrim -av
+ExecStart=/usr/bin/sonic-fstrim
```

Comment:
Where does `sonic-fstrim` get installed?

Reply:
This has been included in sonic-utilities/scripts as part of pull request 
https://github.com/Azure/sonic-utilities/pull/829 

As mentioned, the PR 4214 has dependency on sonic-utilities pull request #829.